introduction.md

```diff
@@ -0,0 +1 @@
+Hello, I'm Roman Mamchyk. I'm 24 years old. I live in Kiev, Ukraine and work as a programmer in EPAM Systems company.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.